ios/ble/Utility/chunker.swift

```diff
@@ -98,9 +105,9 @@ class Chunker {
     }
 
     func isComplete() -> Bool {
-        let isComplete = chunksReadCounter > (totalChunkCount - 1)
+        let isComplete = chunksReadIndex > (totalChunkCount - 1)
```

Comment:
This method is to check if all the chunks have been transferred. So we can simplify the logic to 
chunksReadIndex == totalChunkCount.

Reply:
Yes, its better to change it to `chunksReadIndex >= totalChunkCount`